packages/api/src/router/notif.ts

```diff
@@ -1,55 +1,53 @@
+import { ChatType } from '@prisma/client'
 import { z } from 'zod'
+
 import { createTRPCRouter, publicProcedure } from '../trpc'
-import { ChatType } from '@prisma/client'
 
 export const notifRouter = createTRPCRouter({
   /**
    *  @remarks
    *  This returns all of the messages in order from oldest to newest within a direct message chat
    *  between two users.
-   * 
+   *
    *  @param  id - the id of the direct message chat
    *  @returns an array of message objects
    */
-  getDMsFromChatId: publicProcedure
-    .input(z.object({ id: z.number().int() }))
-    .query(async ({ ctx, input }) => {
-      const { prisma } = ctx
+  getDMsFromChatId: publicProcedure.input(z.object({ id: z.number().int() })).query(async ({ ctx, input }) => {
+    const { prisma } = ctx
 
-      const chat = await prisma.chat.findFirst({
-        where: {
-          type: ChatType.DIRECT,
-          id: input.id,
-        },
-        select: {
-          id: true,
-          users: true,
-        },
-      })
+    const chat = await prisma.chat.findFirst({
+      where: {
+        type: ChatType.DIRECT,
+        id: input.id,
+      },
+      select: {
+        id: true,
+        users: true,
+      },
+    })
 
-      if (chat == undefined) {
-        return []
-      }
+    if (chat == undefined) {
+      return []
```

Comment:
This is great, one thing if I were to beware of is

![image](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/9398066d-04a7-452f-953a-011a7e18da73)

For when you do a query or a mutation and the call doesn't exist data which leads to an early return. You can simply error out here, which will not break our app, but rather inform us through our console (terminal) that the query gets back nothing, it also returns an empty array (or object {} to the client). 

This will help better debug experience in the future, as return an [] will just let the app go on.

A minor suggestion, not a mandatory one.

```
import { TRPCError } from '@trpc/server'
```

Reply:
I have another issue where I am making changes to these APIs. Can I just implement this change in that issue instead of this one?

Reply:
You can do so, yes